Practice/kruglov/home_task_4/PyLec4T5.py

```diff
@@ -0,0 +1,18 @@
+# 5 Interpolation
+
+
+def inter(str1, dict1):
+    str2 = ""
+    for el in str1.split():
+        if el in dict1:
+            str2 += dict1[el]
+        else:
+            str2 += el + " "
+    return str2
+
+
+s2 = "This is r"
```

Comment:
This test example doesn't cover the significant part of the tested code.

Reply:
Fixed